features/publishing_tools.feature

```diff
@@ -1,3 +1,4 @@
+@app-publishing-api
```

Comment:
What does this tagging actually achieve? Do we ever just run a subset of smokey tests by its tags, or does it produce more useful error output, or is this purely for categorisation / local development?

Reply:
Indeed we do: https://github.com/alphagov/govuk-puppet/pull/10387

Reply:
Excellent :) Will we have any corresponding docs on https://docs.publishing.service.gov.uk/?

Reply:
Yeah, that's on my list to do. I'd like to get this in if possible, and address the documentation separately.

I've made a start on that [here](https://github.com/alphagov/smokey/pull/683), if you'd like to review that 😃.